test/studies/prk/stencil/PERFCOMPOPTS

```diff
@@ -0,0 +1 @@
+-sassertNoSlicing
```

Comment:
Noted. I'll try to capture the -sassertNoSlicing performance delta in a smaller performance test, outside of the PRKs.